website/content/docs/concepts/events.mdx

```diff
@@ -143,27 +138,56 @@ JSON for the requested events (one line per event):
 ...
 ```
 
 ## Policies
 
-To subscribe, the `read` capability must be granted by a [policy](/vault/docs/concepts/policies)
-on the `/v1/sys/events/subscribe/{eventType}` path, where `{eventType}` is the event type that will be
-subscribed to. The path may contain wildcards.
+To subscribe, two capabilities must be granted by [policies](/vault/docs/concepts/policies):
 
-An example blanket policy is:
-```hcl
-path "sys/events/subscribe/*" {
-    capabilities = ["read"]
-}
-```
+1. the `read` capability on the `/v1/sys/events/subscribe/{eventType}` path, where `{eventType}` is the event type that will be
+   subscribed to. The path may contain wildcards.
+
+   An example blanket policy is:
+   ```hcl
+   path "sys/events/subscribe/*" {
+       capabilities = ["read"]
+   }
+   ```
 
+1. For events related to secrets, the `list` and `subscribe` capabilities must be granted on the *path of the secret*,
+   and must contain a `subscribe_event_types` entry with the events the subscriber will be allowed to use.
+   For example, to receive events related to the KV secrets engine path, `secret/my-data`, a policy could be:
+
+   ```hcl
+   path "secret/my-data" {
+     capabilities = ["list", "subscribe"]
+     subscribe_event_types = ["*"]
+   }
+   ```
+
+-> Note: if the policy for the *path of the secret* also contains the `read` capability, we **may** include
+   the contents of the secret in the event payload in a future version of Vault.
+
+Policies are continuously evaluated for the WebSocket subscription while it is running.
+These checks are cached for a short period of time for performance reasons, so it is possible that
+events may be sent for a few minutes after a token has been revoked or a policy has been deleted.
 
 ## Supported versions
 
+In Vault 1.15+, events are enabled by default.
+
+For Vault 1.13 and 1.14, events are disabled by default when starting a Vault server and can be
+enabled via the `events.alpha1` [experiment option](/vault/docs/configuration#experiments) in the Vault
+configuration or on the command-line:
+
+```shell-session
+$ vault server -experiment events.alpha1
+```
+
 | Vault Version | Support                                     |
 | ------------- | ------------------------------------------- |
 | <= 1.12       | Not supported                               |
-| 1.13          | Supported (with `events.alpha1` experiment) |
\ No newline at end of file
+| 1.13–1.14     | Supported (with `events.alpha1` experiment) |
+| 1.15+         | Supported                                   |
```

Comment:
Suggest moving the table to before the example command so it's clearer to folks whether or not the command example is relevant to them